.nvmrc

```diff
@@ -1 +1 @@
-v6.11.5
```

Comment:
What do you think about symlinking this file to point to `calypso/.nvmrc`?

Reply:
That makes sense! Is that supported on Windows?

Reply:
> That makes sense! Is that supported on Windows?

I don't think so 🙁 

But…

> Note: nvm does not support Windows (see #284). Two alternatives exist, which are neither supported nor developed by us […]

☝️  https://github.com/creationix/nvm#important-notes

So maybe we don't care about Windows support for `nvm` related stuff?

Reply:
> So maybe we don't care about Windows support for nvm related stuff?

Yup that sounds accurate then if nvm isn't supported. 😄